alerts/system_alerts.libsonnet

```diff
@@ -59,7 +59,7 @@
               severity: 'warning',
             },
             annotations: {
-              message: "Kubernetes API server client '{{ $labels.job }}/{{ $labels.instance }}' is experiencing {{ printf \"%0.0f\" $value }} errors / second.",
+              message: "Kubernetes API server client '{{ $labels.job_name }}/{{ $labels.instance }}' is experiencing {{ printf \"%0.0f\" $value }} errors / second.",
```

Comment:
here `job` is the correct label, as it's not the kubernetes job, but the `job` label of Prometheus